### PR TITLE
Add Heiman HS2SS-E_V03 smart doorbell model

### PR DESCRIPTION
**Pull request by hugokernel**
Hi,
This PR add the support for DoorBell-EF-3.0 from Heiman.

### CODE REVIEW

File: devices/heiman.js [hunk 1]
```diff
@@ -737,6 +737,20 @@ module.exports = [
         },
         exposes: [e.battery(), e.action(['pressed']), e.battery_low(), e.tamper()],
     },
+    {
+        fingerprint: [{modelID: 'DoorBell-EF-3.0', manufacturerName: 'HEIMAN'}],
+        model: 'HS2SS-E_V03',
```
Comment (Koenkk) [inline devices/heiman.js:742]: Is this the correct model? https://www.heimantech.com/manual/index.php?type=detail&id=44 says this is a smart scene switch.

**Comment by hugokernel** [comment]
Yes, I'm sure about the reference.

As you can see on the images, there is only one button. This item is sold as a doorbell.



heiman-bouton-de-sonnette-intelligent-zigbee-sans-carillon

**Comment by Koenkk** [comment]
Thanks!

**Comment by hugokernel** [comment]
Thank you !

**Pull request merged**